torch/_dynamo/symbolic_convert.py

```diff
@@ -1012,8 +1050,13 @@ def CALL_FUNCTION_KW(self, inst):
 
     def LOAD_METHOD(self, inst):
         self.LOAD_ATTR(inst)
-        self.push(self.pop())
-        self.push(None)
+        obj = self.pop()
+        if sys.version_info >= (3, 11):
+            self.PUSH_NULL(inst)
```

Comment:
This assumes that this is never a method?
My understanding from the doc is that if `obj` is a bound method, then we should push `obj` + `TOS`. Otherwise we should push `NULL` + `obj`.

I would actually expect the first case to be more common no?

Reply:
Or if this is working in a special way with the CALL_METHOD below, I guess it needs to be updated because we're not pushing the dummy None anymore.

Reply:
CALL_METHOD is no longer present in 3.11. My understanding is that either `unbound_meth` + `TOS` or `NULL` + `bound_meth` will work, so I just choose the latter out of convenience (so that I don't have to access `__func__` or `__self__`).

Reply:
CALL_METHOD doesn't exist anymore but LOAD_METHOD still does both of these things according to the doc: https://docs.python.org/3.10/library/dis.html#opcode-LOAD_METHOD

In particular, the object level method used for implementing this in the interpr has this doc:
```cpp
/* Specialized version of _PyObject_GenericGetAttrWithDict
   specifically for the LOAD_METHOD opcode.

   Return 1 if a method is found, 0 if it's a regular attribute
   from __dict__ or something returned by using a descriptor
   protocol.

   `method` will point to the resolved attribute or NULL.  In the
   latter case, an error will be set.
*/
int
_PyObject_GetMethod(PyObject *obj, PyObject *name, PyObject **method)
...
```

And the choice of what is pushed to the stack depends on the return value from this function.

I'm surprised that our implementation wouldn't need to do that and can do either.
In particular my understanding is that for methods to actually get `self` as their first argument, we rely on this pushing `self` on top of the stack as the argument loading code should only load the non-self args.

Reply:
Bound methods already have a reference to `self`, so we don't need to include it in args.

Reply:
Ok, so it is indeed because we expect the CALL to call `self` which might be a bound method here (which it can't be in the real bytecode case) then this all works out.
That might deserve a comment but sounds good!